install.md

```diff
@@ -4,7 +4,9 @@
 
 #### [Arch Linux](https://www.archlinux.org)
 
-Podman is available in the AUR through the [libpod package](https://aur.archlinux.org/packages/libpod/)
+```bash
+sudo pacman -S podman
+```
```

Comment:
Does it make sense to keep the old verbiage?  Perhaps these new lines then:

or alternatively, Podman is available in the AUR through the [libpod package](https://aur.archlinux.org/packages/libpod/)

Reply:
The AUR package (`libpod`) does not exist anymore, only the official package (`podman`) is available.

Reply:
Thanks for the info, LGTM.